### ISSUE TITLE
PSR12.Functions.NullableTypeDeclaration false positive when ternary operator used with instanceof

### ISSUE THREAD

**Issue by asika32764** [squizlabs/PHP_CodeSniffer#2791]
This code will return `There must not be a space between the question mark and the type in nullable type declarations`

```php
return $value instanceof static ? '(' . $value . ')' : $value;
```

![8E7EEA1A-7EEF-45E7-8612-51C7869F6D9E](https://user-images.githubusercontent.com/1639206/71638536-7012f680-2c9d-11ea-83d8-a4308d7c3e66.png)

Snuffer version: 3.5.3

**Comment by jrfnl** [comment]
@asika32764 Tested & confirmed as a tokenizer bug.

**Comment by gsherwood** [comment]
Thanks for reporting this. Fix will be in 3.5.4.
